### ISSUE TITLE
'Fill out' feature not working with SNT version 4.0.1

### ISSUE THREAD

**Issue by Maker2000b** [morphonets/SNT#65]
With the new update of SNT, the 'fill out' feature from the Path Manager in SNT doesn't work anymore. Instead of filling a narrow band of similar grey values along the previously drawn paths, it now creates a huge blob that renders this function unusable.

**Comment by carshadi** [comment]
Hi @Maker2000b,

It sounds like the distance threshold setting may be too high. Perhaps try setting it manually to 0.01 - 0.03 ? We increased the initial threshold because for certain images it was not clear the process had started, so the Fill is (intentionally) likely to "overshoot" if the user does not intervene. Which option are you using to set the threshold? We encourage users to set the distance threshold using the interactive "Set by clicking on traced structure" option.  If doing so interactively, it helps to disable the cursor snapping checkbox and to make sure the current image z-slice position is near the desired Path where you click.

If you are still having issues we could move this discussion over to [the forum](https://forum.image.sc/), as this is not technically a bug.

**Comment by Maker2000b** [comment]
Thank you for your reply. The problem is that I started my analysis setting it manually to 0.03 in the old version and that same setting now creates large and unusable overshoot as in the automatic fill setting. In essence it it not reproducible from what I did with the old version of SNT. I am currently trying to finish my data analysis in an older version stored in a computer where Fiji and the SNT plugin was not updated.
